lib/copy-sync/__tests__/copy-sync-dir.test.js

```diff
@@ -27,15 +27,15 @@ describe('+ copySync()', () => {
       src = path.join(TEST_DIR, 'src')
       dest = path.join(TEST_DIR, 'dest')
 
-      fs.mkdirsSync(src)
+      fs.mkdirSync(src)
```

Comment:
I can't speak for @manidlou, but `mkdirs` technically isn't needed, since we know the parent directory exists.